spec/restaurants_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'dishes'
+require 'restaurant'
+
+describe 'restaurant' do
+
+	subject { Restaurant.new }
+
+	it 'creates a restaurant' do
+		expect(subject).to be_kind_of Restaurant
+	end 
+
+	it 'creates a menu' do
+		expect(subject.menu).to 
```

Comment:
Vad ska det här testet göra? Du avslutar ed ett `.to` men ingen matcher eller förväntat resultat.